pkg/apis/eksctl.io/v1alpha5/validation_test.go

```diff
@@ -352,9 +352,12 @@ var _ = Describe("ClusterConfig validation", func() {
 
 	Describe("ssh flags", func() {
 		var (
-			testKeyPath = "some/path/to/file.pub"
-			testKeyName = "id_rsa.pub"
-			testKey     = "THIS IS A KEY"
+			testKeyPath                                 = "some/path/to/file.pub"
```

Comment:
Nit: we didn't have this function before but now we can [use this](https://github.com/weaveworks/eksctl/blob/master/pkg/utils/strings/strings.go#L4) instead of declaring a string variable here and then using it to make a pointer out of it:

```
			It("It fails when the spotAllocationStrategy is not a supported strategy", func() {
				ng.InstancesDistribution.SpotAllocationStrategy = strings.Pointer("unsupported-strategy")

				err := validateInstancesDistribution(ng)
				Expect(err).To(HaveOccurred())
			})
```

Reply:
nice, yeah "strings" makes it much nicer looking